curve strat calculation

rolandpo opened the pull request:
calculate accurate collateral proportions for curve-based strategies, add LUSD share to dashboard

shahthepro reviewed (approved):
Took a quick pass. Looks good to me. After merging, Let's test this on staging before prod though

rolandpo commented:
also removed comp reward tokens from dashboard and strategy endpoint to clean things up. I'm not sure it's relevant before it gets claimed and converted, especially since it's the only rewards token shown
